README.md

```diff
@@ -56,16 +56,32 @@ Each release of Dapr CLI includes various OSes and architectures. These binary v
 ```
 $ dapr init
 ⌛  Making the jump to hyperspace...
-↗   Downloading binaries and setting up components...
-✅  Success! Dapr is up and running
+↓  Downloading binaries and setting up components...
+removing archive /Users/msundar/.dapr/daprd_darwin_amd64.tar.gz
+↙  Downloading binaries and setting up components...
+installing Dapr to /usr/local/bin
+
+removing extracted binary /Users/msundar/.dapr/daprd
+✅  Downloading binaries and setting up components...
```

Comment:
Change the text to something like (in the code as well): downloaded and configured